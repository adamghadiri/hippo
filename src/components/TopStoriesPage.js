import React from "react";
import Story from "./Story";

export default () => (
  <div className="content-container">
    <h3 className="list-header">Popular updates</h3>
    <div className="list-body">
      {[...Array(10).keys()].map(rank => (
        <Story key={rank} showRank={true} rank={rank} />
      ))}
    </div>
  </div>
);
        