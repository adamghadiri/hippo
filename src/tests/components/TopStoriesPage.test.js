import React from "react";
import { shallow } from "enzyme";
import TopStoriesPage from "../../components/TopStoriesPage";

test("should render top stories page correctly", () => {
  const wrapper = shallow(<TopStoriesPage />);
  expect(wrapper).toMatchSnapshot();
});
