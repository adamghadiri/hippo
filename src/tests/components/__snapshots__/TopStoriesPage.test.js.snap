// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render top stories page correctly 1`] = `
<div
  className="content-container"
>
  <h3
    className="list-header"
  >
    Popular updates
  </h3>
  <div
    className="list-body"
  >
    <ConnectFunction
      key="0"
      rank={0}
      showRank={true}
    />
    <ConnectFunction
      key="1"
      rank={1}
      showRank={true}
    />
    <ConnectFunction
      key="2"
      rank={2}
      showRank={true}
    />
    <ConnectFunction
      key="3"
      rank={3}
      showRank={true}
    />
    <ConnectFunction
      key="4"
      rank={4}
      showRank={true}
    />
    <ConnectFunction
      key="5"
      rank={5}
      showRank={true}
    />
    <ConnectFunction
      key="6"
      rank={6}
      showRank={true}
    />
    <ConnectFunction
      key="7"
      rank={7}
      showRank={true}
    />
    <ConnectFunction
      key="8"
      rank={8}
      showRank={true}
    />
    <ConnectFunction
      key="9"
      rank={9}
      showRank={true}
    />
  </div>
</div>
`;
